src/PendingBets.tsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import { observer } from 'mobx-react-lite';
+import { Heading, Button } from 'rebass';
+import styled from 'styled-components';
+import { store } from './store';
+import { Table } from './Table';
+import { Prime } from './types';
+
+const StakeButton = styled(Button)`
+  padding: 0 8px 0 12px;
+  text-align: center;
+  height: 30px;
+  font-size: 14px;
+  background-color: transparent;
+  border: 1px solid #0000ff;
+  color: #000000;
+
+  & + & {
```

Comment:
It's better to set
```
margin-right: 10px;
&:last-of-type{
    margin-right: 0;
}
```

Reply:
Why it's better?

Reply:
Less technical debt? 🙄
Also, from my experience, setting right and bottom margins prevents you from hairpulling in the future :)